src/main/java/uk/gov/hmcts/dts/fact/repositories/CourtRepository.java

```diff
@@ -17,6 +17,10 @@
     @Query("UPDATE Court c SET c.info = :info, c.infoCy = :infoCy WHERE c.slug in :slugs")
     void updateInfoForSlugs(@Param("slugs") List<String> slugs, @Param("info") String info, @Param("infoCy") String infoCy);
 
+    @Modifying(clearAutomatically = true)
```

Comment:
Just nooby thoughts on this: 

1) Does the mapit data have any relationships in terms of the persistence context? 
2) It's weird that a .save() wouldn't work here for updating the mapit data, I think if you do a getOne query, and then set the lat and lon to be different, and then save again, it does an update rather than an insert behind the scenes, but up to you really! This way is fine too 

Feel free to resolve this one on your own if you are happy with thoughts, otherwise can play around with using the jpa method maybe to update rather than a custom query

Reply:
I was doing the other way initially but then change it to this way as I thought it is slightly more efficient. Using the other way, you first needs to call the courtRepository to retrieve the court entity, then update the lat long before saving (like below). As we have the court slug from the input so I just use it to update using custom query. I don't mind either way as the overhead is really minor

final Court courtEntity = courtRepository.findBySlug(slug)
    .orElseThrow(() -> new NotFoundException(slug));
courtEntity.setLat(lat);
courtEntity.setLon(lon);
courtEntity.save(courtEntity);